Presentations/Week06/Cohort2/index.html

```diff
@@ -0,0 +1,615 @@
+<!DOCTYPE html>
```

Comment:
Hiya, thanks for adding this! Can you also upload this? https://github.com/meenakshi-kushwaha/advR_functions/blob/master/index.Rmd :)